resources/js/Pages/Results.vue

```diff
@@ -1,28 +1,51 @@
 <template>
     <div class="page-container results-page">
         <Head title="Mismatch Finder - Results" />
-        <section id="results" v-if="Object.keys(results).length">
-            <section class="item-mismatches" v-for="(mismatches, item, idx) in results" :key="idx">
+        <div v-if="Object.keys(results).length < item_ids.length" class="not-found">
+            <div id="top-button">
+                <wikit-button
+                    variant="normal"
+                    type="neutral"
+                    native-type="button"
+                    @click.native="returnHome"
+                >
+                    &#x2190; {{ $i18n('refine-items-selection') }}
+                </wikit-button>
+            </div>
+            <section id="description-section">
+                <h2 class="h4">{{ $i18n('results-page-title') }}</h2>
+                <p id="about-description" >{{ $i18n('results-page-description') }}</p>
+            </section>
```

Comment:
I don't think these UI elements are in scope for this task, they should be tackled at [T290861](https://phabricator.wikimedia.org/T290861). Either or, and since you already added the code, I think that they should also show up when we get full results.

Reply:
Done.